examples/hello/hdf5Writer/helloHDF5Writer_nompi.cpp

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
This isn't really a no-mpi version of the example.  Best to leave it out until we fix the serial hdf5 vs parallel hdf5 issue.